run.py

```diff
@@ -372,15 +367,21 @@ def reindex_failed(app):
                         default=False,
                         help='sends bibcodes to augment affilation pipeline, works with --filename')
     parser.add_argument('--solr-collection',
-                       dest='solr_collection',
-                       default='collection2',
-                       action='store',
-                       help='name of solr collection, currently only used by rebuild collection')
+                        dest='solr_collection',
+                        default='collection2',
+                        action='store',
+                        help='name of solr collection, currently only used by rebuild collection')
     parser.add_argument('-x',
                         '--rebuild-collection',
                         action='store_true',
                         default=False,
                         help='Will send all solr docs for indexing to another collection; by purpose this task is synchronous. You can send the name of the collection or the full url to the solr instance incl http via --solr-collection')
+    parser.add_argument('--priority',
+                        dest='priority',
+                        action='store',
+                        default=0,
+                        type=int,
+                        help='priority to use in queue, typically cron jobs use a high priority')
```

Comment:
Can we include in the description if higher number means higher priority? Sometimes being priority 1 seems to indicate higher priority, sometimes is just the highest number. Better to clarify what is in this case so that we avoid misinterpretations. Also, is there a range of valid? Can they be negative?